ui/lib/core/addon/components/form-field.js

```diff
@@ -75,6 +75,11 @@ export default class FormFieldComponent extends Component {
     this.showInput = !!modelValue;
   }
 
+  get hasRadioSubText() {
+    // check to see if any of the possibleValues has a subText for radio buttons.
```

Comment:
it might be worth including to clarify that it's specifically for editType = 'radio'
```suggestion
    // for 'radio' editType, check to see if any of the possibleValues has a subText
```